pkg/reconciler/brokercell/resources/deployments.go

```diff
@@ -127,6 +127,12 @@ func MakeRetryDeployment(args RetryArgs) *appsv1.Deployment {
 
 // deploymentTemplate creates a template for data plane deployments.
 func deploymentTemplate(args Args, containers []corev1.Container) *appsv1.Deployment {
+	annotation := map[string]string{
+		"sidecar.istio.io/inject": strconv.FormatBool(args.AllowIstioSidecar),
+	}
+	if args.RolloutRestartTime != "" {
```

Comment:
Looks like removing this condition is still fine.

Reply:
Thanks for the comment! If removing this condition and there is no annotation for rollout restart time added the Brokercell resource, the deployment will have `"events.cloud.google.com/RestartedAt": ""`. It is functional logically, but not that meaningful, so we want to avoid this case.

Reply:
I am fine with either way.